src/nupic/algorithms/Connections.cpp

```diff
@@ -360,6 +360,8 @@ const SynapseData &Connections::dataForSynapse(Synapse synapse) const {
 
 UInt32 Connections::segmentFlatListLength() const { return (UInt32)segments_.size(); }
 
+UInt32 Connections::synapseFlatListLength() const { return (UInt32)synapses_.size(); }
```

Comment:
nit, use size_t /Size according to #357

Reply:
I'd rather change to `size_t` all at once in one PR, instead of scattering the changes throughout many PRs.

Reply:
please make this size_t, Conn now uses that in other places already